src/render/components/workflows.js

```diff
@@ -23,44 +23,55 @@ const isReverse = (flowA, flowB) => {
 const workflowsList = (thisItem, workflows, metaIndex) => {
   const itemAnchor = (itemName) => workflowItemAnchor(itemName, metaIndex);
 
-  const filteredFlows = [];
+  const labeledFlows = {};
+  const pushLabeledFlow = (baseKey, base, itemKey, item) => {
+    if (!labeledFlows[baseKey]) {
+      labeledFlows[baseKey] = {base, items: {}};
+    }
+    labeledFlows[baseKey].items[itemKey] = item;
+  };
```

Comment:
This function builds up similar workflows as they are iterated over. The use of keyed objects for the base and items allows deduplication to happen at the same time.